src/olympia/conf/dev/settings.py

```diff
@@ -111,9 +111,7 @@
 
 REDIRECT_URL = 'https://outgoing.stage.mozaws.net/v1/'
 
-CLEANCSS_BIN = 'cleancss'
```

Comment:
`cleancss` and `uglifyjs` are not needed after deploy. `Dockerfile.deploy` already sets a path for them at deploy time.